internal/app/machined/pkg/system/services/registryd.go

```diff
@@ -0,0 +1,80 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Perhaps this went to wrong branch: this is missing from #9722

Reply:
Thanks! Missed.